### DISCUSSION TITLE
Laravel + Statamic - control panel?

### DISCUSSION THREAD

**Discussion by AimForNaN** [statamic/cms#5997] [Q&A] [answered]
Let's say we [install laravel](https://laravel.com/docs/9.x#installation-via-composer), then we [install Statamic into our existing laravel](https://statamic.dev/installing/laravel#install-statamic). Now, how would we go about setting up our laravel app to work with Statamic without having to setup a user and without having to use the control panel? The documentation appears to assume the control panel throughout and that your app is already fully configured for Statamic after installation, while briefly mentioning that [the control panel "can be" optional](https://statamic.dev/core-concepts#the-control-panel-can-be-optional). Am I forced to look through the source code of the control panel?

**Comment by afonic** [comment]
That depends on what you mean by "work with Statamic". You can definately work without the CP (like a headless CMS), which usually means populating the needed files in `/content` and `/resources` by other means. Statamic is a flat file CMS, so all you need to start working are a couple of `yaml` and `md` files.

Example: You can create a Collection and it's Blueprint manually and then use the `Entry` facade to create entries inside your Laravel application. If this sounds similar to what you're trying to achieve, check the available Repositories: https://statamic.dev/reference/repositories

**Reply by AimForNaN** [reply]
That's what I thought, but it doesn't seem to happen so smoothly.

Let's say I have the following collection:
**/content/collections/blog.yaml**
```yaml
template: blog.blade.php
route: '/blog/{slug}'
```

And the following entry:
**/content/collections/blog/test.md**
```md
---
id: ca22c1ef-0e8b-46fa-abc4-afddec9bc8a9
title: Test
---

test
```

And the template at **/resources/views/blog.blade.php**.

Why would navigating to `/blog/test` exit out with the error?
```
View [blog.blade.php] not found.
```

**Comment by AimForNaN** [comment, marked as answer]
Nevermind, I got it all working now.

**Comment by jackmcdade** [comment]
Take the .blade.php out of the route definition in the tank config.
